src/VisualStudio/Core/Def/Implementation/AbstractEditorFactory.cs

```diff
@@ -303,7 +303,7 @@ private void FormatDocumentCreatedFromTemplate(IVsHierarchy hierarchy, uint item
             var formattedText = addedDocument.GetTextSynchronously(cancellationToken).WithChanges(formattedTextChanges);
 
             // Ensure the line endings are normalized. The formatter doesn't touch everything if it doesn't need to.
-            var targetLineEnding = documentOptions.GetOption(FormattingOptions.NewLine);
+            var targetLineEnding = documentOptions.GetOption(FormattingOptions.NewLine)!;
```

Comment:
should we have GetRequiredOption?

Reply:
It will no longer return null once #41627 is completed